test/test_util_type_list.cpp

```diff
@@ -21,7 +21,7 @@
 
 #include <type_traits>
 #include <map>
-#include <realm/util/type_list.hpp>
+#include <realm/util/tuple.hpp>
```

Comment:
That was the plan. Unfortunately util::Tuple it is still needed by "safe_int_ops", so more needs to be done